bedrock/firefox/templatetags/helpers.py

```diff
@@ -173,6 +173,60 @@ def download_firefox(ctx, channel='release', platform='all',
     return jinja2.Markup(html)
 
 
+@library.global_function
+@jinja2.contextfunction
+def download_firefox_thanks(ctx, dom_id=None, locale=None, alt_copy=None, button_class=None,
+                            locale_in_transition=False, download_location=None):
+    """ Output a simple "download firefox" button that only points to /download/thanks/
+
+    :param ctx: context from calling template.
+    :param dom_id: Use this string as the id attr on the element.
+    :param locale: The locale of the download. Default to locale of request.
+    :param alt_copy: Specifies alternate copy to use for download buttons.
+    :param button_class: Classes to add to the download button, contains size mzp-t-xl by default
+    :param locale_in_transition: Include the page locale in transitional download link.
+    :param download_location: Specify the location of download button for
+            GA reporting: 'primary cta', 'nav', 'sub nav', or 'other'.
+    """
+
+    channel = 'release'
+    locale = locale or get_locale(ctx['request'])
+    funnelcake_id = ctx.get('funnelcake_id', False)
+    dom_id = dom_id or 'download-button-thanks'
+    transition_url = '/firefox/download/thanks/'
+    version = firefox_desktop.latest_version(channel)
```

Comment:
This is not something we should fix in this PR, but the `version` parameter to the `get_download_url` isn't even used in the function anymore. It's a vestige of the old way of generating direct download links before we only used the `*-latest` bouncer aliases. This doesn't hurt anything, but you could also just pass in `None` for `version` and it would work the same.